packages/core/dist/components/MonthPicker/MonthPicker.js

```diff
@@ -153,7 +158,11 @@ var MonthPicker = exports.MonthPicker = function (_React$PureComponent) {
       }
 
       if (!this.isControlled) {
-        this.setState({ selectedMonths: [] });
+        this.setState({
+          selectedMonths: [],
+          selectAllPressed: false,
+          clearAllPressed: true
```

Comment:
and these

Reply:
Same as previous comment. I won't check in the rendered files anymore: https://github.com/CMSgov/design-system/pull/235/files/1433a682fef50adfa8cc35c69dfafb8c04e048f2..9647e024ca0d412a12b45839bade5d704c08bf06#diff-6c5bf996b9de0d202168b671faa4c0da